uisdk/src/androidTest/java/com/karhoo/uisdk/booking/braintree/BraintreeBookingFlowTests.kt

```diff
@@ -340,20 +318,20 @@ class BookingFlowTests : Launch {
             clickPickUpAddressField()
         }
         address {
-            search(TestData.SEARCH_ADDRESS)
+            search(SEARCH_ADDRESS)
             clickBakerStreetResult()
         }
         booking {
-            sleep()
+            shortSleep()
         } result {
-            selectedPickupAddressIsVisible(address = TestData.SELECTED_ADDRESS)
+            selectedPickupAddressIsVisible(address = SELECTED_ADDRESS)
         }
         booking {
-            sleep(MEDIUM)
+            shortSleep()
```

Comment:
mediumSleep()?

Reply:
I've tested it and a short sleep is enough